cpp/ps0001/ps0001_solv.cpp

```diff
@@ -4,12 +4,18 @@ using namespace std;
 
 int main()
 {
-    int n1,n2,n3,sum=0;
-    for (int i =1 ; i <=25 ; i++)
+    // declared 3 variables to store user inputs 0.5/1
+    unsigned short n1,n2,n3,sum=0;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`